ebuild-writing/error-handling/text.xml

```diff
@@ -35,6 +35,10 @@ Decent error handling is important because:
 <title>The <c>die</c> Function</title>
 <body>
 
+<note>
+You should use <c>die</c> on almost all external commands in ebuilds.
+</note>
+
```

Comment:
Move the note to the end of the section.

Also, it says almost the same as the last paragraph of the section ("It's best to use || die too often than too little."), so maybe that paragraph could be dropped.